### ISSUE TITLE
January Bug : month=0 ?

### ISSUE THREAD

**Issue by GTorreil** [milesmcc/shynet#188]
Happy new year everyone ! 

I'm affraid it might be "New Year's Bugs Time". 
I just came back to work today and found the shynet instance to be down. It seems there's a `now.replace(day=1, month=now.month - 1)` somewhere, which sets `month=0` now that we're in January. 

I don't know the inner logic of this part so I have no idea how to fix it... 

- **Steps to reproduce :** GET /dashboard/
- **Trace :** see below

```
ERROR Internal Server Error: /dashboard/
shynet_1              | Traceback (most recent call last):
shynet_1              |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
shynet_1              |     response = get_response(request)
shynet_1              |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
shynet_1              |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
shynet_1              |   File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
shynet_1              |     return self.dispatch(request, *args, **kwargs)
shynet_1              |   File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/mixins.py", line 71, in dispatch
shynet_1              |     return super().dispatch(request, *args, **kwargs)
shynet_1              |   File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 98, in dispatch
shynet_1              |     return handler(request, *args, **kwargs)
shynet_1              |   File "/usr/local/lib/python3.9/site-packages/django/views/generic/list.py", line 157, in get
shynet_1              |     context = self.get_context_data()
shynet_1              |   File "/usr/src/shynet/dashboard/views.py", line 36, in get_context_data
shynet_1              |     data = super().get_context_data(**kwargs)
shynet_1              |   File "/usr/src/shynet/dashboard/mixins.py", line 64, in get_context_data
shynet_1              |     data["date_ranges"] = self.get_date_ranges()
shynet_1              |   File "/usr/src/shynet/dashboard/mixins.py", line 50, in get_date_ranges
shynet_1              |     "start": now.replace(day=1, month=now.month - 1),
shynet_1              | ValueError: month must be in 1..12
```

Thanks for having a look :)

**Comment by haaavk** [comment]
Thanks for bug report. 
This bug is fixed in new release (v0.12.0). 
Just pull new docker image. 
For more info #185

**Comment by GTorreil** [comment]
My bad, sorry for the duplicate. 
Pulling now ! :)
